include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2639,16 +2551,14 @@ DoFCellAccessor<dimension_, space_dimension_, level_dof_access>::get_dof_values(
   Assert(values.size() == this->get_dof_handler().n_dofs(),
          typename DoFCellAccessor::ExcVectorDoesNotMatch());
 
-  const types::global_dof_index *cache =
-    dealii::internal::DoFAccessorImplementation::Implementation::get_cache_ptr(
-      this->dof_handler,
-      this->present_level,
-      this->index(),
-      this->get_fe().n_dofs_per_cell());
+  boost::container::small_vector<types::global_dof_index, 27> dof_indices(
```

Comment:
How did you come up with 27?

Reply:
I assume this is for quadratic hex elements (HEX27) in 3D.

Reply:
Could we make this value a static constant somewhere? That would make it simpler to change in the future.

Reply:
Would be great to have a comment here explaining this.

Reply:
I agree, I will add a new static constexpr variable to the dof accessor class for this. The choice 27 is indeed motivated by getting the most common low-order cases fast (and for the larger ones, the dof index does not matter much). I wanted to have up to scalar Q_2 (27 in 3D) and vector Q_1 (24) covered, plus 2D Stokes with Taylor-Hood (22).

Reply:
I now added an alias to keep the definition in a single place, and added a comment on what the motivation for that was.